src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_IsPatternOperator.cs

```diff
@@ -52,6 +52,11 @@ public new void Free()
 
             private void AddConjunct(BoundExpression test)
             {
+                // When in error recovery, the generated code doesn't matter.
+                if (test.Type.IsErrorType())
```

Comment:
I thought we didn't even get to lowering if there are errors in initial binding. Am I missing something?

Reply:
This is for recovering from an error during lowering, such as string's operator== missing.

---
In reply to: [389189200](https://github.com/dotnet/roslyn/pull/42207#discussion_r389189200) [](ancestors = 389189200)